blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DMetricsCFPreparedStatements.java

```diff
@@ -97,22 +98,28 @@ private DMetricsCFPreparedStatements() {
         //
         insertToMetricsPreaggrFullStatement = session.prepare(
                 String.format(INSERT_KEY_COLUMN_VALUE_TTL_FORMAT,
-                        CassandraModel.CF_METRICS_PREAGGREGATED_FULL_NAME));
+                        CassandraModel.CF_METRICS_PREAGGREGATED_FULL_NAME))
+                .setConsistencyLevel(ConsistencyLevel.ONE); // needed by maven cassandra plugin
```

Comment:
For our purpose, LOCAL_ONE and ONE is the same. LOCAL_ONE goes only to local DC. ONE can go to non-local DC. We only have one DC. So it should be the same.